MIQ-Booking-Assistance/miq-assistance.js

```diff
@@ -142,12 +148,17 @@ async function findAvailability(page) {
     return await page.evaluate((myDates, testDates, findAnyDate, checkedCount) => {
         const nextElem = document.getElementById('form_next');
         nextElem.scrollIntoView();
-
-        const availableDateElements = [...document.querySelectorAll('[class$="_item"] :not(.no)')].filter(d => d.ariaLabel);
+        
+        const prefix = document.querySelector('.mo-6 > div').getAttribute('class')
```

Comment:
I don't think it's guranteed that the class will be `mo-6` I've seen other values here.

Reply:
@sebjamesdev are you sure there are other values for the class than `mo-6` ? what other values have you found?
 this is by the way a class for the calendar container.

Reply:
Sorry, my bad that was the logged out portal calendar.